python/pyiceberg/catalog/hive.py

```diff
@@ -233,24 +299,47 @@ def create_table(
             ValueError: If the identifier is invalid
         """
         database_name, table_name = self.identifier_to_database_and_table(identifier)
-        current_time_millis = int(time.time())
+        seconds_since_epoch = int(time.time() * 1000)
+
+        location = self._resolve_table_location(location, database_name, table_name)
+
+        metadata_location = f"{location}/metadata/{uuid.uuid4()}.metadata.json"
+        metadata = TableMetadataV2(
+            location=location,
+            schemas=[schema],
+            current_schema_id=schema.schema_id,
+            partition_specs=[partition_spec],
+            default_spec_id=partition_spec.spec_id,
+            sort_orders=[sort_order],
+            default_sort_order_id=sort_order.order_id,
+            properties=properties or {},
+            last_updated_ms=seconds_since_epoch,
+            last_column_id=schema.highest_field_id,
+            last_partition_id=max(field.field_id for field in partition_spec.fields)
+            if partition_spec.fields
+            else DEFAULT_LAST_PARTITION_ID,
+        )
+        io = load_file_io({**self.properties})
```

Comment:
If we're always using `self.properties` then I think it makes sense to have an `io` instance for the catalog. That can be used here rather than creating a new one per table.

Reply:
I actually think that we should mix in the `properties` from the table in there as well. This way you can override this on a table level.